app/src/db/migrations/20240327000000_001-bring-forward.ts

```diff
@@ -0,0 +1,31 @@
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  return Promise.resolve()
+    .then(() =>
+      knex.schema.alterTable('submission', function (table) {
+        table.dropColumn('bring_forward_date');
+      })
+    )
+    .then(() =>
+      knex.schema.alterTable('note', function (table) {
+        table.timestamp('bring_forward_date', { useTz: true });
+        table.text('bring_forward_state').nullable();
```

Comment:
fyi. you don't need to add nullable(). fields are nullable by default on column creation